renderer/d3d/pls_render_context_d3d_impl.cpp

```diff
@@ -716,8 +716,13 @@ ID3D11RenderTargetView* PLSRenderTargetD3D::targetRTV()
 {
     if (m_targetRTV == nullptr && m_targetTexture != nullptr)
     {
+        D3D11_RENDER_TARGET_VIEW_DESC desc;
+        ZeroMemory(&desc, sizeof(desc));
```

Comment:
I know `ZeroMemory` is the "standard" way of doing this, but in other places we just do `desc{}`.

Could we do `D3D11_RENDER_TARGET_VIEW_DESC desc{};` instead, without the `ZeroMemory` for consistency?